src/Tizen.NUI/src/internal/WebView/WebContext.cs

```diff
@@ -690,5 +725,15 @@ private void OnPasswordDataListAcquired(IntPtr alist)
             }
             passwordDataListAcquiredCallback?.Invoke(passwordDataList);
         }
+
+        private void OnHttpRequestIntercepted(IntPtr interceptor)
+        {
+            if (httpRequestInterceptor != null)
+            {
+                httpRequestInterceptor.Dispose();
+            }
+            httpRequestInterceptor = new WebHttpRequestInterceptor(interceptor, true);
+            httpRequestInterceptedCallback?.Invoke(httpRequestInterceptor);
```

Comment:
`public delegate bool HttpRequestInterceptedCallback(WebHttpRequestInterceptor interceptor);`

callback returning bool value but you do not use it

Reply:
Done. I changed it from bool to void.
Thanks.